Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c35/translation.txt

```diff
@@ -1924,13 +1924,13 @@
 お尻の中……ぐぐぐって広がる感覚……あれって、こういうエッチ以外じゃなかなか感じられんよね……=This feeling of... having my butt be stretched apart... It's not a feeling you can really experience without having anal, is it...
 お尻の中……まだ余韻が残っとるよ……=I can still feel it... in my butt...
 私の尻穴……オチンポちゃんにこじ開けられちゃったん……=My asshole... it's been wrenched open completely by your dick...
-入り口が無理矢理広げられる感覚やったよ……あ、お尻やけん、出口かもしれんけど……はぅぅ……=I felt like the entrance was being wrenched apart willy-nilly... ah, since it's my butt, maybe "outlet" would be more apt... hauu...
+入り口が無理矢理広げられる感覚やったよ……あ、お尻やけん、出口かもしれんけど……はぅぅ……=I felt like the entrance was being wrenched apart willy-nilly... ah, since it's my butt, maybe outlet" would be more apt... hauu..."
```

Comment:
```suggestion
入り口が無理矢理広げられる感覚やったよ……あ、お尻やけん、出口かもしれんけど……はぅぅ……=I felt like the entrance was being wrenched apart willy-nilly... ah, since it's my butt, maybe "outlet" would be more apt... hauu...
```